documentation/src/main/asciidoc/topics/code_examples/HotRodClientInstance.java

```diff
@@ -0,0 +1,12 @@
+org.infinispan.client.hotrod.configuration.ConfigurationBuilder cb
+      = new org.infinispan.client.hotrod.configuration.ConfigurationBuilder();
+cb.tcpNoDelay(true)
+  .statistics()
+      .enable()
+      .jmxEnable()
+      .jmxDomain(org.infinispan)
+      .jmxName()
```

Comment:
`jmxName` requires a String param. I think it can be any value of significance to the user.

Reply:
yeah I wasn't sure and didn't want to add something arbitrarily. removed it.